### ISSUE TITLE
2019-11-18 tc012 M1SS estimated concentration dataset doesn't include any data for AUCs requiring interpolation or extrapolation

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#24]
tc012 M1SS single tau testcase:

```
> j <- parameter_required("^AUC[0-9]+?_[0-9]+?", names(r), simplify=FALSE)
> r[, c("SDEID", names(j))]
       SDEID   AUC0_12   AUC0_8  AUC13_24   AUC0_10  AUC12_30
1  130509330 106.28661 68.54661 101.01331  90.44661 112.81663
2  147299665 127.75800 74.57500  23.49161 101.57500  73.56096
3   24280050 121.70093 73.21000  22.86411  97.93000  69.57968
4  246409536  95.85319 60.81320  98.71463  80.49320 116.48335
5  246953363 132.22544 70.38500  35.31717 101.62500 101.83405
6  249265153 105.81029 72.25441  91.54349  91.66109 105.24380
7  283369224 114.04271 71.71271  99.91402  95.98271 115.81051
8  292748221 122.16927 79.47927 112.93158 103.56927 130.96473
9  297583299 132.20000 76.39500  26.74885 105.83036  78.93559
10 328228072  88.32373 59.80799  77.48153  76.38155  89.02996
11 330747542 112.24156 56.16500  29.08473  84.02500  88.21482
12 331625416 122.09137 71.06137 126.62559  99.92137 148.21804
13 331909809 138.43017 79.18000  30.52975 108.94000  91.42768
14 361955746 122.78796 72.57998  21.54994  99.22797  67.70626
15 372645306 145.30759 86.97000  34.52197 117.55135  96.67961
16 392032034 120.81805 70.88500  33.59945  96.32397  91.33066
17  45831434 135.53617 84.53981  30.15169 110.98868  85.08787
18 494439910 112.34286 76.28286 105.79389  97.01286 122.01738
19 543244472 122.89782 78.21380 108.80205 102.91218 120.43644
20 545913303 131.41914 72.60000  33.73834 101.18000 100.20367
21 579885120 118.70500 58.86500  35.66342  86.36500 106.65215
22 695874499 104.99736 65.78736  95.46428  87.80736 106.38086
23 702415421 151.49158 95.08500  34.66523 124.71463  92.83962
24 705396261 151.38648 83.33500  46.54758 117.72680 127.06918
25  70745987 109.22692 67.51494 101.45221  91.01704 109.01992
26 708723045 162.47091 86.09000  44.12660 125.28559 126.86082
27 716555376 137.28371 79.19000  35.46160 108.74820  98.07116
28  72354714 165.66456 92.14500  56.67040 128.68500 148.45348
29 777074699 148.96822 86.15000  31.92311 117.23000  95.83193
30 783385801 121.90605 79.12605 109.52263 104.47605 129.27205
31 797620983 129.24303 75.30303 129.72168 106.08303 140.93127
32 856727921 142.12160 84.61160 139.59919 117.28160 158.58041
33 895085608 128.76625 79.23625 126.03772 107.67625 144.30093
34 897885073 132.62947 81.23947 131.13516 109.79947 147.78610
35 916516839 121.50282 76.98282 113.49268 102.06282 129.42216
36 982849649 125.99427 78.86427 110.91988 105.68427 129.27825
```

where AUC13_24 and AUC12_30 are included to test interpolation/extrapolation. However the estimated concentration dataset doesn't include any values for interpolation or extrapolation.

```
   PKDATAROWID     SDEID   TIME  CEST_KEL CEST_INT CEST_EXT CEST_C0 CEST_TLAST
1      26908252  24280050 12.000  5.467556       NA       NA      NA         NA
2      26908253  24280050 18.000  2.696850       NA       NA      NA         NA
3      26908254  24280050 24.000  1.330210       NA       NA      NA   1.330210
4      26908341  45831434  8.000  7.594357       NA       NA      NA         NA
5      26908342  45831434 10.000  6.382756       NA       NA      NA         NA
6      26908343  45831434 12.000  5.364453       NA       NA      NA         NA
7      26908344  45831434 18.000  3.184764       NA       NA      NA         NA
8      26908345  45831434 24.000  1.890727       NA       NA      NA   1.890727
9      26908924  70745987  2.000  5.638244       NA       NA      NA         NA
10     26908925  70745987  4.000  5.382876       NA       NA      NA         NA
11     26908926  70745987  6.000  5.139075       NA       NA      NA         NA
12     26908927  70745987  9.000  4.793919       NA       NA      NA         NA
13     26908928  70745987 12.000  4.471946       NA       NA      NA         NA
14     26908929  70745987 12.333  4.437567       NA       NA      NA         NA
15     26908930  70745987 12.667  4.403351       NA       NA      NA         NA
16     26908931  70745987 13.000  4.369500       NA       NA      NA         NA
17     26908932  70745987 13.500  4.319161       NA       NA      NA         NA
18     26908933  70745987 14.000  4.269402       NA       NA      NA         NA
19     26908934  70745987 16.000  4.076032       NA       NA      NA         NA
20     26908935  70745987 18.000  3.891420       NA       NA      NA         NA
21     26908936  70745987 21.000  3.630061       NA       NA      NA         NA
22     26908937  70745987 24.000  3.386256       NA       NA      NA   3.386256
23     26908291  72354714 12.000  9.540110       NA       NA      NA         NA
24     26908292  72354714 18.000  5.836801       NA       NA      NA         NA
25     26908293  72354714 24.000  3.571053       NA       NA      NA   3.571053
26     26908951 130509330 13.500  8.226134       NA       NA      NA         NA
27     26908952 130509330 14.000  7.703235       NA       NA      NA         NA
28     26908953 130509330 16.000  5.923561       NA       NA      NA         NA
29     26908954 130509330 18.000  4.555043       NA       NA      NA         NA
30     26908955 130509330 21.000  3.071546       NA       NA      NA         NA
31     26908956 130509330 24.000  2.071197       NA       NA      NA   2.071197
32     26908018 147299665 12.000  5.948676       NA       NA      NA         NA
33     26908019 147299665 18.000  2.791242       NA       NA      NA         NA
34     26908020 147299665 24.000  1.309709       NA       NA      NA   1.309709
35     26908532 246409536 13.000  7.993709       NA       NA      NA         NA
36     26908533 246409536 13.500  7.556942       NA       NA      NA         NA
37     26908534 246409536 14.000  7.144040       NA       NA      NA         NA
38     26908535 246409536 16.000  5.706041       NA       NA      NA         NA
39     26908536 246409536 18.000  4.557492       NA       NA      NA         NA
40     26908537 246409536 21.000  3.253213       NA       NA      NA         NA
41     26908538 246409536 24.000  2.322197       NA       NA      NA   2.322197
42     26908317 246953363 12.000  7.453740       NA       NA      NA         NA
43     26908318 246953363 18.000  3.934901       NA       NA      NA         NA
44     26908319 246953363 24.000  2.077272       NA       NA      NA   2.077272
45     26908866 249265153  1.500  7.301496       NA       NA      NA         NA
46     26908867 249265153  2.000  7.158180       NA       NA      NA         NA
47     26908868 249265153  4.000  6.612501       NA       NA      NA         NA
48     26908869 249265153  6.000  6.108420       NA       NA      NA         NA
49     26908870 249265153  9.000  5.423425       NA       NA      NA         NA
50     26908871 249265153 12.000  4.815245       NA       NA      NA         NA
51     26908872 249265153 12.333  4.752090       NA       NA      NA         NA
52     26908873 249265153 12.667  4.689577       NA       NA      NA         NA
53     26908874 249265153 13.000  4.628071       NA       NA      NA         NA
54     26908875 249265153 13.500  4.537230       NA       NA      NA         NA
55     26908876 249265153 14.000  4.448172       NA       NA      NA         NA
56     26908877 249265153 16.000  4.109081       NA       NA      NA         NA
57     26908878 249265153 18.000  3.795840       NA       NA      NA         NA
58     26908879 249265153 21.000  3.370176       NA       NA      NA         NA
59     26908880 249265153 24.000  2.992246       NA       NA      NA   2.992246
60     26908513 283369224 13.000  9.017636       NA       NA      NA         NA
61     26908514 283369224 13.500  8.412782       NA       NA      NA         NA
62     26908515 283369224 14.000  7.848499       NA       NA      NA         NA
63     26908516 283369224 16.000  5.945310       NA       NA      NA         NA
64     26908517 283369224 18.000  4.503626       NA       NA      NA         NA
65     26908518 283369224 21.000  2.969232       NA       NA      NA         NA
66     26908519 283369224 24.000  1.957609       NA       NA      NA   1.957609
67     26908893 292748221 13.000  9.929105       NA       NA      NA         NA
68     26908894 292748221 13.500  9.299885       NA       NA      NA         NA
69     26908895 292748221 14.000  8.710541       NA       NA      NA         NA
70     26908896 292748221 16.000  6.703709       NA       NA      NA         NA
71     26908897 292748221 18.000  5.159233       NA       NA      NA         NA
72     26908898 292748221 21.000  3.483295       NA       NA      NA         NA
73     26908899 292748221 24.000  2.351772       NA       NA      NA   2.351772
74     26908108 297583299 10.000  7.328109       NA       NA      NA         NA
75     26908109 297583299 12.000  5.880090       NA       NA      NA         NA
76     26908110 297583299 18.000  3.037806       NA       NA      NA         NA
77     26908485 328228072  1.000  6.393740       NA       NA      NA         NA
78     26908486 328228072  1.500  6.263443       NA       NA      NA         NA
79     26908487 328228072  2.000  6.135801       NA       NA      NA         NA
80     26908488 328228072  4.000  5.650722       NA       NA      NA         NA
81     26908489 328228072  6.000  5.203992       NA       NA      NA         NA
82     26908490 328228072  9.000  4.599235       NA       NA      NA         NA
83     26908491 328228072 12.000  4.064757       NA       NA      NA         NA
84     26908492 328228072 12.333  4.009399       NA       NA      NA         NA
85     26908493 328228072 12.667  3.954633       NA       NA      NA         NA
86     26908494 328228072 13.000  3.900775       NA       NA      NA         NA
87     26908495 328228072 13.500  3.821282       NA       NA      NA         NA
88     26908496 328228072 14.000  3.743409       NA       NA      NA         NA
89     26908497 328228072 16.000  3.447465       NA       NA      NA         NA
90     26908498 328228072 18.000  3.174918       NA       NA      NA         NA
91     26908499 328228072 21.000  2.805961       NA       NA      NA         NA
92     26908500 328228072 24.000  2.479879       NA       NA      NA   2.479879
93     26908304 330747542 12.000  6.886258       NA       NA      NA         NA
94     26908305 330747542 18.000  3.393008       NA       NA      NA         NA
95     26908306 330747542 24.000  1.671808       NA       NA      NA   1.671808
96     26908817 331625416 13.000 12.769752       NA       NA      NA         NA
97     26908818 331625416 13.500 10.631781       NA       NA      NA         NA
98     26908819 331625416 14.000  8.851760       NA       NA      NA         NA
99     26908057 331909809 12.000  7.001066       NA       NA      NA         NA
100    26908058 331909809 18.000  3.485191       NA       NA      NA         NA
101    26908059 331909809 24.000  1.734958       NA       NA      NA   1.734958
102    26908042 361955746  8.000  7.027679       NA       NA      NA         NA
103    26908043 361955746 10.000  6.206793       NA       NA      NA         NA
104    26908044 361955746 12.000  5.481793       NA       NA      NA         NA
105    26908238 372645306 10.000  7.457668       NA       NA      NA         NA
106    26908239 372645306 12.000  6.344770       NA       NA      NA         NA
107    26908240 372645306 18.000  3.907100       NA       NA      NA         NA
108    26908082 392032034 10.000  6.498894       NA       NA      NA         NA
109    26908083 392032034 12.000  5.705948       NA       NA      NA         NA
110    26908084 392032034 18.000  3.861828       NA       NA      NA         NA
111    26908837 494439910 13.500  8.242931       NA       NA      NA         NA
112    26908838 494439910 14.000  7.749338       NA       NA      NA         NA
113    26908839 494439910 16.000  6.053358       NA       NA      NA         NA
114    26908840 494439910 18.000  4.728551       NA       NA      NA         NA
115    26908841 494439910 21.000  3.264568       NA       NA      NA         NA
116    26908842 494439910 24.000  2.253841       NA       NA      NA   2.253841
117    26908960 543244472  1.000  7.597843       NA       NA      NA         NA
118    26908961 543244472  1.500  7.479809       NA       NA      NA         NA
119    26908962 543244472  2.000  7.363608       NA       NA      NA         NA
120    26908963 543244472  4.000  6.916580       NA       NA      NA         NA
121    26908964 543244472  6.000  6.496690       NA       NA      NA         NA
122    26908965 543244472  9.000  5.914163       NA       NA      NA         NA
123    26908966 543244472 12.000  5.383868       NA       NA      NA         NA
124    26908967 543244472 12.333  5.328019       NA       NA      NA         NA
125    26908968 543244472 12.667  5.272584       NA       NA      NA         NA

```

**Comment by jhhughes256** [comment]
This issue is related to existing issue #7 and also occurs for tc011, tc013, tc014 and tc015.

**Comment by jhhughes256** [comment]
### Update
This issue has been fixed as of 066fa5d.
